framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -645,14 +646,17 @@ def set_noise_equation_params(self, **data):
         return rendering_rules.to_dict()
 
     @staticmethod
-    def _get_form_url_after_monitors():
+    def _get_form_url_after_monitors(loading=False):
         burst_config = common.get_from_session(common.KEY_BURST_CONFIG)
         operation_group = dao.get_operationgroup_by_id(burst_config.operation_group_id)
 
         if operation_group is not None:
-            return SimulatorWizzardURLs.SETUP_PSE_URL
-        else:
-            return SimulatorWizzardURLs.LAUNCH_SIMULATION_URL
+            if loading is False:
+                return SimulatorWizzardURLs.SETUP_PSE_URL
+            else:
+                return SimulatorWizzardURLs.LAUNCH_PSE_URL
+
+        return SimulatorWizzardURLs.LAUNCH_SIMULATION_URL
```

Comment:
I think this is were your problems come from. LAUNCH_PSE_URL should not be used as an URL to go to from the NEXT button/action. And by NEXT action I refer to an action that gives us the next fragment in our wizzard.
In your case, the fragment with buttons Launch or SetupPSE should have as action the SETUP_PSE_URL, because that one represents the NEXT action, that gives us the next fragment.
Launch action is called in another manner from JS part.